core/src/main/scala/kafka/coordinator/transaction/TransactionMetadata.scala

```diff
@@ -323,7 +323,7 @@ private[transaction] class TransactionMetadata(val transactionalId: String,
           }
 
         case CompleteAbort | CompleteCommit => // from write markers
-          info(s"transit start ${transitMetadata.txnStartTimestamp}")
+          debug(s"Transaction for $transactionalId begin transit to $toState at ${transitMetadata.txnStartTimestamp}")
```

Comment:
We have used the word "transit" intentionally in place of "transition"  in some cases. Actually the log message below seems sufficient to cover this message, so I'm just going to remove this log.